airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/migrate_to_manifest_only/manifest_component_transformer.py

```diff
@@ -4,10 +4,61 @@
 
 import copy
 import typing
-from typing import Any, Mapping
+import logging
+from typing import Any, Mapping, Set, Type
+from .declarative_component_schema import *
```

Comment:
I just copied the existing models so they could be accessible to the tool. It's hacky and not sustainable in anything but the short term since the models are now static. Happy to implement a better approach, but this does work as a short-term unblocker for migrations.